pkgs/desktops/gnome-3/default.nix

```diff
@@ -21,36 +21,6 @@ lib.makeScope pkgs.newScope (self: with self; {
 
   maintainers = with pkgs.lib.maintainers; [ lethalman jtojnar hedning worldofpeace ];
 
-  corePackages = with gnome3; [
```

Comment:
@jtojnar I know you said you don't consider this public api, but it's possible someone could have used them. Should these become `throw`s to use the options?

Reply:
Do not see any downsides.